public/js/views/management/add-pay-method.jsx

```diff
@@ -33,7 +35,7 @@ export default class AddPayMethod extends Component {
         <CardForm
           submissionErrors={this.props.cardSubmissionErrors}
           submitPrompt={gettext('Add')}
-          handleCardSubmit={(card) => this.handleCardSubmit(card)}
+          handleCardSubmit={this.handleCardSubmit.bind(this)}
```

Comment:
Could you make the handleCardSubmit a fat arrow func then you probably don't need the bind.

e.g:
```
handleCardSubmit = (creditCard, processingId) => {

}
```